src/main/java/seedu/duke/finance/CommandFinanceDel.java

```diff
@@ -16,11 +16,21 @@ public String execute() {
 
     @Override
     public String help() {
-        return "The format of input to delete a log is: finance delLog index";
+        return "Syntax: finance delLog <index>";
     }
 
     public int validate(UserInput ui) {
-        if (ui.getCategory().equals("finance") && ui.getCommand().equalsIgnoreCase("dellog")) {
+        if (ui.getCategory().equals("finance") && ui.getCommand().equalsIgnoreCase("dellog")
+                || ui.getCommand().equalsIgnoreCase("del")
+                || ui.getCommand().equalsIgnoreCase("d")) {
+            if (ui.getArg("") == null || ui.getArg("").equals("")) {
+                return ARGUMENT_ERR;
+            }
+            try {
+                Integer.parseInt(ui.getArg(""));
+            } catch (NumberFormatException e) {
+                return ARGUMENT_ERR;
+            }
```

Comment:
I think just use the try part below is enough. Because when it equals "", it also throws NumberFormatException.

Reply:
The same when it is null

Reply:
Great suggestion. I pushed a new update to implement the change you suggested.